int8 operation_mode
---
int8 result